src/commands/createNewProject/PythonProjectCreator.ts

```diff
@@ -159,12 +159,22 @@ export class PythonProjectCreator extends ScriptProjectCreatorBase {
         const funcInitPython: string = 'func init ./ --worker-runtime python';
         switch (process.platform) {
             case Platform.Windows:
-                await cpUtils.executeCommand(ext.outputChannel, this.functionAppPath, `${await this.getVenvActivatePath(Platform.Windows)} && ${funcInitPython}`);
+                await cpUtils.executeCommand(ext.outputChannel, this.functionAppPath, `${await this.getVenvActivatePath(Platform.Windows)} && ${funcInitPython} && pip install ptvsd`);
                 break;
             case Platform.MacOS:
             default:
-                await cpUtils.executeCommand(ext.outputChannel, this.functionAppPath, `source ${await this.getVenvActivatePath(Platform.MacOS)} && ${funcInitPython}`);
+                await cpUtils.executeCommand(ext.outputChannel, this.functionAppPath, `source ${await this.getVenvActivatePath(Platform.MacOS)} && ${funcInitPython} && pip install ptvsd`);
```

Comment:
Maybe "pip install ptvsd" should be a part of creating the virtual environment, rather than in "requirements.txt". We will have to add logic to re-create the venv if it's not there anyways

Reply:
Yeah... I was hoping you wouldn't say that.  I put it in here because it already had switch cases based on Platform, but you're right regarding comment 2.